api/tests.py

```diff
@@ -44,6 +44,38 @@ def setUp(self):
                                 "terminal"], timestamp=self.demo["timestamp"],
                             token=self.demo["token"],
                             status=self.demo["status"])
+    def setUp2(self):
```

Comment:
Why do we need another setUp?

Reply:
It is for the function test_get_one_demo_without_id_and user_id, as it returns all the demo present in the database.I thought it would be better if we add more than one demo in database so we could easily check if it is working properly.

Reply:
We can do that in the other setUp as well right?

Reply:
Oh you mean creating another demo object in the setUp,yeah absolutely.But that would create 2 demo object for each function,but i guess that wont be a problem.